### ISSUE TITLE
CI

### ISSUE THREAD

**Issue by kekyo** [kekyo/IL2C#18]

**Comment by kekyo** [comment]
* Use appveyor
* And travis?

**Comment by kekyo** [comment]
AppVeyor or SDK bug? 

```
Build started
git clone -q --branch=master https://github.com/kekyo/IL2C.git C:\projects\il2c
git checkout -qf 2e0ee10ed2765e8a407447b9aef027e78b9e9cd3
dotnet restore
C:\projects\il2c\tests\Win32\Win32.Target\Win32.Target.vcxproj(28,3): error MSB4019: The imported project "C:\Microsoft.Cpp.Default.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
C:\projects\il2c\tests\WDM\WDM.Target\WDM.Target.vcxproj(47,3): error MSB4019: The imported project "C:\Microsoft.Cpp.Default.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
C:\projects\il2c\tests\Win32\Win32.Target\Win32.Target.vcxproj : warning NU1503: Skipping restore for project 'C:\projects\il2c\tests\Win32\Win32.Target\Win32.Target.vcxproj'. The project file may be invalid or missing targets required for restore. [C:\projects\il2c\il2c.sln]
C:\projects\il2c\tests\WDM\WDM.Target\WDM.Target.vcxproj : warning NU1503: Skipping restore for project 'C:\projects\il2c\tests\WDM\WDM.Target\WDM.Target.vcxproj'. The project file may be invalid or missing targets required for restore. [C:\projects\il2c\il2c.sln]
  Restoring packages for C:\projects\il2c\IL2C.Core.Tests\IL2C.Core.Tests.csproj...
  Restoring packages for C:\projects\il2c\IL2C.Core\IL2C.Core.csproj...
  Installing Mono.Cecil 0.10.0-beta7.
  Installing Microsoft.TestPlatform.TestHost 15.5.0.
  Installing Microsoft.CodeCoverage 1.0.3.
  Installing NUnit 3.9.0.
  Installing Microsoft.NET.Test.Sdk 15.5.0.
  Installing NUnit3TestAdapter 3.9.0.
  Generating MSBuild file C:\projects\il2c\IL2C.Core.Tests\obj\IL2C.Core.Tests.csproj.nuget.g.props.
  Generating MSBuild file C:\projects\il2c\IL2C.Core.Tests\obj\IL2C.Core.Tests.csproj.nuget.g.targets.
  Restore completed in 1.38 sec for C:\projects\il2c\IL2C.Core.Tests\IL2C.Core.Tests.csproj.
  Restoring packages for C:\projects\il2c\IL2C\IL2C.csproj...
  Generating MSBuild file C:\projects\il2c\IL2C.Core\obj\IL2C.Core.csproj.nuget.g.props.
  Generating MSBuild file C:\projects\il2c\IL2C.Core\obj\IL2C.Core.csproj.nuget.g.targets.
  Restore completed in 1.65 sec for C:\projects\il2c\IL2C.Core\IL2C.Core.csproj.
  Restoring packages for C:\projects\il2c\tests\WDM\WDM.Code\WDM.Code.csproj...
  Installing runtime.any.System.Diagnostics.Tracing 4.1.0.
  Installing runtime.win-x86.Microsoft.NETCore.DotNetAppHost 2.0.0.
  Installing runtime.win.System.Diagnostics.Debug 4.0.11.
  Installing runtime.any.System.Globalization 4.0.11.
  Installing runtime.win7.System.Private.Uri 4.0.1.
  Installing runtime.any.System.Text.Encoding.Extensions 4.0.11.
  Installing runtime.win-x86.Microsoft.NETCore.DotNetHostResolver 2.0.0.
  Installing runtime.any.System.Runtime.InteropServices 4.1.0.
  Installing runtime.any.System.Resources.ResourceManager 4.0.1.
  Installing System.Private.Uri 4.0.1.
  Installing runtime.any.System.Reflection.Primitives 4.0.1.
  Installing runtime.any.System.Threading.Tasks 4.0.11.
  Installing runtime.any.System.Text.Encoding 4.0.11.
  Installing runtime.any.System.Runtime.Handles 4.0.1.
  Installing runtime.any.System.IO 4.1.0.
  Installing runtime.any.System.Runtime 4.1.0.
  Installing runtime.win-x86.Microsoft.NETCore.DotNetHostPolicy 2.0.0.
  Installing runtime.win.System.Runtime.Extensions 4.1.0.
  Installing runtime.any.System.Reflection 4.1.0.
  Installing runtime.win.System.IO.FileSystem 4.0.1.
  Installing runtime.any.System.Collections 4.0.11.
  Installing runtime.win-x86.Microsoft.NETCore.App 2.0.0.
  Generating MSBuild file C:\projects\il2c\tests\WDM\WDM.Code\obj\WDM.Code.csproj.nuget.g.props.
  Generating MSBuild file C:\projects\il2c\tests\WDM\WDM.Code\obj\WDM.Code.csproj.nuget.g.targets.
  Restore completed in 3.64 sec for C:\projects\il2c\tests\WDM\WDM.Code\WDM.Code.csproj.
  Restoring packages for C:\projects\il2c\tests\Win32\Win32.Code\Win32.Code.csproj...
  Generating MSBuild file C:\projects\il2c\tests\Win32\Win32.Code\obj\Win32.Code.csproj.nuget.g.props.
  Generating MSBuild file C:\projects\il2c\tests\Win32\Win32.Code\obj\Win32.Code.csproj.nuget.g.targets.
  Restore completed in 21.24 ms for C:\projects\il2c\tests\Win32\Win32.Code\Win32.Code.csproj.
  Restoring packages for C:\projects\il2c\tests\WioLTE\WioLTE.Code\WioLTE.Code.csproj...
  Generating MSBuild file C:\projects\il2c\tests\WioLTE\WioLTE.Code\obj\WioLTE.Code.csproj.nuget.g.props.
  Generating MSBuild file C:\projects\il2c\tests\WioLTE\WioLTE.Code\obj\WioLTE.Code.csproj.nuget.g.targets.
  Restore completed in 22.04 ms for C:\projects\il2c\tests\WioLTE\WioLTE.Code\WioLTE.Code.csproj.
  Generating MSBuild file C:\projects\il2c\IL2C\obj\IL2C.csproj.nuget.g.props.
  Generating MSBuild file C:\projects\il2c\IL2C\obj\IL2C.csproj.nuget.g.targets.
  Restore completed in 5.34 sec for C:\projects\il2c\IL2C\IL2C.csproj.
msbuild "C:\projects\il2c\il2c.sln" /verbosity:minimal /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
Microsoft (R) Build Engine version 15.5.180.51428 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.
  IL2C.Core -> C:\projects\il2c\IL2C.Core\bin\Debug\netstandard2.0\IL2C.Core.dll
  IL2C -> C:\projects\il2c\IL2C\bin\Debug\netcoreapp2.0\IL2C.dll
  IL2C.Core -> C:\projects\il2c\IL2C.Core\bin\Debug\net45\IL2C.Core.dll
  IL2C -> C:\projects\il2c\IL2C\bin\Debug\net45\IL2C.exe
  IL2C.Core.Tests -> C:\projects\il2c\IL2C.Core.Tests\bin\Debug\net45\IL2C.Core.Tests.dll
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(1601,5): error : Project 'C:\projects\il2c\IL2C\IL2C.csproj' targets 'netcoreapp2.0;net45'. It cannot be referenced by a project that targets '.NETStandard,Version=v1.3'. [C:\projects\il2c\tests\Win32\Win32.Code\Win32.Code.csproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(1601,5): error : Project 'C:\projects\il2c\IL2C\IL2C.csproj' targets 'netcoreapp2.0;net45'. It cannot be referenced by a project that targets '.NETStandard,Version=v1.3'. [C:\projects\il2c\tests\WioLTE\WioLTE.Code\WioLTE.Code.csproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(1601,5): error : Project 'C:\projects\il2c\IL2C\IL2C.csproj' targets 'netcoreapp2.0;net45'. It cannot be referenced by a project that targets '.NETStandard,Version=v1.3'. [C:\projects\il2c\tests\WDM\WDM.Code\WDM.Code.csproj]
Command exited with code 1
```

**Comment by kekyo** [comment]
Hmm...
* AppVeyor success build but test failed (a lot of test have failed already known)
* Circle CI failed because il2c.sln related project contains net45 configuration, Circle CI build only .NET Core 2.0 on docker.

**Comment by kekyo** [comment]
* Circle CI fixed at 998107a06b83b3d3b2f8535f629f23a492c88d67
* Appvayor failed at unit tests (maybe not appveyors problem.
